teamscale-client/src/main/kotlin/com/teamscale/client/JsonUtils.kt

```diff
@@ -52,18 +60,18 @@ object JsonUtils {
 	 */
 	@Throws(JsonProcessingException::class)
 	@JvmStatic
-	fun <T> deserializeList(json: String, elementClass: Class<T>): List<T> =
+	inline fun <reified T> deserializeList(json: String): List<T> =
 		OBJECT_MAPPER.readValue(
-			json, OBJECT_MAPPER.typeFactory.constructCollectionLikeType(MutableList::class.java, elementClass)
+			json, OBJECT_MAPPER.typeFactory.constructCollectionLikeType(MutableList::class.java, T::class.java)
 		)
 
 	/**
 	 * Serializes an object into its JSON representation.
 	 */
 	@JvmStatic
 	@Throws(JsonProcessingException::class)
-	fun serialize(value: Any): String =
-		OBJECT_MAPPER.writeValueAsString(value)
+	fun Any.serialize(): String =
```

Comment:
Now that it is an extension function the name might be a bit too unspecific.

```suggestion
	fun Any.serializeToJson(): String =
```